### PR TITLE
test responsive page in rootlayout and pipline layout

### PR DESCRIPTION
**Pull request by jessieyu777**
test layout with tailwind tag css

### CODE REVIEW

File: packages/playground/src/layouts/RepoLayout.tsx [hunk 1]
```diff
@@ -7,10 +7,10 @@ const RepoLayout: React.FC = () => {
 
   return (
     <div>
-      <header style={{ background: '#333', padding: '20px', height: '56px' }}>
+      <header className="p-5 h-16 bg-black">
```
Comment (abhinavrastogi-harness) [inline packages/playground/src/layouts/RepoLayout.tsx:10]: let's not use absolute colors like `black`. use semantic colors instead, like `bg-background`

File: packages/playground/src/layouts/RepoLayout.tsx [hunk 2]
```diff
@@ -7,10 +7,10 @@ const RepoLayout: React.FC = () => {
...
         <h1>Repository {repoId}</h1>
       </header>
-      <header style={{ background: '#222', padding: '20px' }}>
+      <header className="p-5 h-16" style={{ lineHeight: 1}}>
```
Comment (abhinavrastogi-harness) [inline packages/playground/src/layouts/RepoLayout.tsx:13]: do we need to define the `lineHeight`? if so, let's not use inline styles.. use TW classes instead

File: packages/playground/src/layouts/RootLayout.tsx [hunk 3]
```diff
@@ -9,9 +9,9 @@ const RootLayout: React.FC = () => {
   const showNavbar = !hideNavbarPaths.includes(location.pathname)
 
   return (
-    <div className="bg-background flex">
+    <div className="bg-background" style={{ display: 'flex' }}>
```
Comment (abhinavrastogi-harness) [inline packages/playground/src/layouts/RootLayout.tsx:12]: add tw class `flex` instead of inline style

Comment (jessieyu777) [reply packages/playground/src/layouts/RootLayout.tsx]: solved to the flex tag in tailwind

File: packages/playground/src/layouts/RepoLayout.tsx (hunk 2 above)
Comment (jessieyu777) [reply packages/playground/src/layouts/RepoLayout.tsx]: deleted line height in advance

File: packages/playground/src/layouts/RepoLayout.tsx (hunk 1 above)
Comment (jessieyu777) [reply packages/playground/src/layouts/RepoLayout.tsx]: Solved from bg-black to bg-background

**Pull request merged**